utils/utils.go

```diff
@@ -22,18 +22,12 @@ func OpenInBrowser(url string) error {
 }
 
 func RenderMarkdown(text string) (string, error) {
-	style := "notty"
-	// TODO: make color an input parameter
-	if isColorEnabled() {
-		style = "auto"
-	}
-
 	// Glamour rendering preserves carriage return characters in code blocks, but
 	// we need to ensure that no such characters are present in the output.
 	text = strings.ReplaceAll(text, "\r\n", "\n")
 
 	tr, err := glamour.NewTermRenderer(
-		glamour.WithStandardStyle(style),
+		glamour.WithEnvironmentConfig(),
```

Comment:
Apart from losing `notty` support, the switch to `WithEnvironmentConfig` opens us back to https://github.com/charmbracelet/glamour/issues/72 right after we fixed it in #1402.

What do you think about this logic instead:
- if color is disabled: `WithStandardStyle("notty")`
- if GLAMOUR_STYLE is set to a simple string (no slashes): `WithStandardStyle(os.Getenv("GLAMOUR_STYLE"))`
- if GLAMOUR_STYLE looks like a path (e.g. has slashes): `WithStylePath(os.Getenv("GLAMOUR_STYLE"))`

Reply:
I think I'd rather fix the root problem in Glamour than pollute your code base with temporary workarounds. Lemme make a PR to fix Glamour. Looks simple enough.

Reply:
PR made: https://github.com/charmbracelet/glamour/pull/73

Reply:
PR merged.